### PR TITLE
Partially revert "update build"

### LINKED ISSUE #113: Diagrams bug
Initial error:
```
klighd-vscode.setLanguageClient command called with invalid arguments.
Please refer to the documentation for reference about the correct usage.
```
After that:
```
command 'klighd-vscode.diagram.open' not found
```

### PR DESCRIPTION
**Pull request by petervdonovan**
This partially reverts commit f85abfc1bc5b634ca4bfa6286017ef859afc8f03.

This fixes #113.

Thank you to @soerendomroes for helping us find the source of the problem.

### CODE REVIEW

**Comment by lhstrh** [comment]
Looks like the fix introduced another problem...

File: .github/actions/build/action.yml
Comment (lhstrh) [inline .github/actions/build/action.yml:1]: Why did this have to be changed? We use `prepare-build-env` almost everywhere, so it should just work...

**Review by lhstrh — COMMENTED** [review]
What's the nature of the changes in the known-good texts?

**Comment by petervdonovan** [comment]
> Why did this have to be changed? We use prepare-build-env almost everywhere, so it should just work...

There is an error due to the CWD being `vscode-lingua-franca` instead of `lingua-franca` during the cleanup process at the end. See [this test failure (in `main`) for example](https://github.com/lf-lang/vscode-lingua-franca/actions/runs/4999552536/jobs/8956051352).


> What's the nature of the changes in the known-good texts?

I updated the lingua-franca submodule, so the content of the tests changed.

**Review by lhstrh — APPROVED** [review]

**Comment by petervdonovan** [comment]
Probably the Windows error was related to the caching improvements. It's the only recent change that I was aware of.

**Pull request merged**